Includes new .rmdoc file information for USB-Web-interface and the specific API call (#56)
Eeems-Org/remarkable.guide: src/tech/usb-web-interface.rst

  curl \
    -I "http://10.11.99.1/download/$guid/placeholder"

``GET http://10.11.99.1/download/{guid}/rmdoc``
-----------------------------------------------

> Download the raw notebook file(.rmdoc) for a specific document.(From v3.9)

Reviewer: ```suggestion
Download the raw notebook archive for a specific document. This was added in v3.9.
```
